### PR TITLE
Feature/#5 candidate list view

### PR DESCRIPTION
**Pull request by jimitpatel9**

### CODE REVIEW

**Comment by jackofseattle** [comment]
You need to add .idea to your .gitignore file

we don't check in editor configs.

File: client/vivace/list/template.html [hunk 1]
```diff
@@ -1,3 +1,46 @@
...
+          <md-divider ng-if="!$last"/>
+       </div>
+      </md-list>
+    </div>
+    <md-button
+        href="#/add/new"
+        class="md-fab md-primary md-hue-2 add-candidate"
+        aria-label="Add candidate"
+        id="btn-add">
+      <md-icon>person_add</md-icon>
```
Comment (jshor) [inline client/vivace/list/template.html]: this doesn't seem to work (the icon doesn't show up)

File: client/vivace/list/controller.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-module.exports = function() {
-  // use candidateService.all() to get a list of all candidates
-  this.status = 'red';
+module.exports = function(dataStoreService) {
+  this.users = dataStoreService.all();
```
Comment (jshor) [inline client/vivace/list/controller.js]: need to pass collectionName (`candidates`) as argument to this function

Comment (jimitpatel9) [inline client/vivace/list/controller.js]: okay

**Comment by jimitpatel9** [comment]
added .idea to gitignore

File: client/vivace/list/template.html (hunk 1 above)
Comment (jimitpatel9) [inline client/vivace/list/template.html]: Icon added

**Comment by jackofseattle** [comment]
LGTM - rebase though

**Pull request merged**